### DISCUSSION TITLE
FIT Export

### DISCUSSION THREAD

**Discussion by ovahead** [mytourbook/mytourbook#930] [Q&A] [answered]
I really can't find a FIT export possibility. Is there none?

**Comment by FJBDev** [comment, marked as answer]
MT currently supports exporting tours in the following extensions : GPX, TCX, MT

[The documentation](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/export-tour) is not up-to-date as it is missing the new [`.MT` extension added](https://github.com/mytourbook/mytourbook/issues/493) in the 22.3 release

**Comment by ovahead** [comment]
Thanks!
Is an export as FIT files intended for the future?

**Reply by FJBDev** [reply]
Not that I know of (unless @wolfgang-ch  knows more than me ?) but feel free to create a feature request.
As for now, I believe there are tools to convert GPX/TCX to FIT such as the [fit file repair tool](https://www.fitfilerepairtool.info/)

**Reply by wolfgang-ch** [reply]
It is not planned to export MT tours in the FIT format

**Reply by ovahead** [reply]
> It is not planned to export MT tours in the FIT format

What a pity! ;)

Sure there is the possibility to convert a GPX (export) with fit file repair tool and maybe others. But e.g. FFRT isn't free at all.
Im not really aware if a GPX file could handle all data from a speed meter as well (heart rate and so on)... Wasn't GPX not only for GPS data?

**Reply by FJBDev** [reply]
> Im not really aware if a GPX file could handle all data from a speed meter as well (heart rate and so on)... Wasn't GPX not only for GPS data?

GPX can handle additional data such as heart rate, cadence, etc....
See example below:

```
 <trkpt lat="45.229606" lon="-121.699863">
            <ele>1203.0</ele>
            <time>2015-05-31T16:51:15Z</time>
            <extensions>
               <gpxtpx:TrackPointExtension>
                  <gpxtpx:hr>93</gpxtpx:hr>
                  <gpxtpx:cad>97</gpxtpx:cad>
               </gpxtpx:TrackPointExtension>
               <gpxdata:distance>2.453</gpxdata:distance>
            </extensions>
         </trkpt>
```

**Comment by ovahead** [comment]
Thanks!

**Reply by mff73** [reply]
A feature request already exists.
https://github.com/mytourbook/mytourbook/issues/845
